recipes/server.rb

```diff
@@ -66,3 +66,13 @@
 sysctl_param 'vm.swappiness' do
   value 0
 end
+
+cookbook_file '/usr/local/bin/mysql-accounting' do
+  source 'mysql-accounting'
+  mode '0755'
+end
```

Comment:
You know, I ran the actual script as an informal test, so I find the setup persuasively better.